classes/class-p4-master-site.php

```diff
@@ -792,6 +792,101 @@ public function register_header_metabox() {
 				'preview_size' => 'large',
 			]
 		);
+
+		// P4 Datalayer/Campaign fields.
+		$p4_campaign_fields = new_cmb2_box(
+			[
+				'id'           => $prefix . 'campaign_fields',
+				'title'        => __( 'Campaign information (dataLayer)', 'planet4-master-theme-backend' ),
+				'object_types' => [ 'page' ],
+				'closed'       => true,  // Keep the metabox closed by default.
+			]
+		);
+
+		$campaign_options = [
+			0                                    => __( '- Select Campaign -', 'planet4-engagingnetworks' ),
```

Comment:
please change this also to `planet4-master-theme-backend`,
same for lines 846, 867